From 85a1f164b2c27da06a4af579d9a25c80f404698b Mon Sep 17 00:00:00 2001
From: Gio <thelookouts@hotmail.com>
Date: Fri, 30 Dec 2022 17:23:50 -0500
Subject: [PATCH] Update 60-sensor.hwdb

Updated DBook D10 / Juno Tablet. Before the bvr (bios version) was hard coded to v1.05. This will read all versions

(cherry picked from commit 8d885b4477ef544093a617027a7a36d34c803479)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 6c26b73060..afb5a79c25 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -362,8 +362,8 @@ sensor:modalias:acpi:INVN6500*:dmi:*svnDell*:pnVenue10Pro5055:*
 # DERE
 #########################################
 
-# DBook D10
-sensor:modalias:acpi:KIOX000A*:dmi:bvnAmericanMegatrendsInternational,LLC.:bvrJP2V1.05:bd04/27/2022:br1.5:efr1.3:svnDefaultstring:*
+# DBook D10 / Juno Tablet
+sensor:modalias:acpi:KIOX000A*:dmi:bvnAmericanMegatrendsInternational,LLC.:bvrJP2V*:svnDefaultstring:*
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
 
 #########################################
